src/components/form/Select.js

```diff
@@ -70,19 +59,9 @@ StyledSelect.propTypes = {
     touched: PropTypes.bool.isRequired,
```

Comment:
`error` propType is missing. but as they're wrapped with the `meta` object, both `error` and `touched` propTypes need to be replaced with a meta propType. I guess we can outsource that to the same file where I put the formInput propType